src/modules/main/sidebar/Sidebar.tsx

```diff
@@ -32,16 +34,28 @@ const getActiveLinkIdx = (pathname: string, links: SidebarLink[]) => {
   });
 };
 
-export const Sidebar = ({ basePath }: SidebarProps) => {
+export const Sidebar = ({ basePath, isSidebarOpen, changeSidebarState }: SidebarProps) => {
   const { pathname } = useLocation();
 
   const activeLinkIdx = getActiveLinkIdx(pathname.replace(basePath, ''), sidebarLinks);
 
   return (
     <aside className={csx.sidebar}>
       <div className={csx.sidebarContent}>
-        <figure className={csx.logo}>
-          <Logo />
+        <figure className={csx.logo} >
+          {isSidebarOpen 
+            ? 
+              <div style={{
+                display: 'flex',
+                alignItems: 'center'
+              }}>
+                <img height="30px" src={IMGS_PATH + '/Logo.png'} />
+                <p style={{marginLeft:"10px"}}>Jupi.io</p>
+                <p style={{marginLeft:"10px"}} onClick={()=>changeSidebarState()}><Close /></p>
+              </div>
+
+
+            : <Menu onClick={()=>changeSidebarState()} />}
```

Comment:
Replace () => changeSidebarState() - to toggleOpen - in every place without callback

Reply:
Callback removed